src/getKeyCodeDefinitions.ts

```diff
@@ -0,0 +1,36 @@
+import { keyCodeDefinitions } from "./keyCodeDefinitions";
+
+export type Key = keyof typeof keyCodeDefinitions;
```

Comment:
just moved this around and tweaked slightly so both `realPress` and `realType` can use this